source/common/filesystem/posix/filesystem_impl.cc

```diff
@@ -160,6 +160,14 @@ bool InstanceImplPosix::illegalPath(const std::string& path) {
       absl::StartsWith(canonical_path.rc_, "/proc")) {
     return true;
   }
+#ifdef FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION
+  // Allow temporary files in OSS-Fuzz to prevent failures running (h1/h2)_capture_fuzz_test.
+  // Ideally this would be TestEnvironment::temporaryPath() but this avoids depending on test
+  // libraries.
+  if (absl::StartsWith(canonical_path.rc_, "/mnt")) {
+    return true;
+  }
+#endif
```

Comment:
Does this mean that the fuzzer code won't be able to access files in _/mnt_?
Could this impact fuzz tests that may need to access external files under the _/mnt_ directory?

Reply:
OMG thanks for pointing that out. The fix was meant to do the opposite.